lib/SingleSelleckt.js

```diff
@@ -179,35 +179,56 @@ _.extend(SingleSelleckt.prototype, {
         this.selectItem(item);
     },
 
-    _filterItems: function(items, term){
-        var matchTerm;
+    _filterItems: function(items, term) {
+        var matchTerm, reduced;
 
-        if(!term){
+        if (!term) {
             return items;
         }
 
         matchTerm = term.toLowerCase();
 
-        return _.reduce(items, function(memo, item){
+        reduced = _.reduce(items, function(memo, item) {
             var matchIndex;
+            var group = 'byName';
+            var matchStart;
+            var matchEnd;
 
             if (!item.value) {
                 return memo;
             }
 
             matchIndex = item.label.toLowerCase().indexOf(matchTerm);
-
-            if(matchIndex === -1){
-                return memo;
+            matchStart = matchIndex;
+            matchEnd = matchIndex + (term.length - 1);
+
+            if (matchIndex === -1 && item.aliases && item.aliases.length > 0) {
+                // try the aliases
+                matchIndex = _.reduce(item.aliases, function(index, alias) {
+                    if (index === -1) {
+                        return String(alias).toLowerCase().indexOf(matchTerm);
+                    } else {
+                        return index;
+                    }
+                }, -1);
+
+                if (matchIndex !== -1) {
+                    group = 'byAlias';
+                    matchStart = matchEnd = 0;
```

Comment:
Did you see [my previous comment](https://github.com/BrandwatchLtd/selleckt/pull/85#discussion_r39973497) about this line?